### DISCUSSION TITLE
Export a convenient `styled` function for quick use

### DISCUSSION THREAD

**Discussion by peduarte** [stitchesjs/stitches#301] [Ideas]
I've been chatting with @StephenHaney about being able to import `styled` directly:
```jsx
import styled from '@stitches/react';

const Box = styled('div', {});
```

This would be the equivalent of calling the `createStyled` function with an empty config:
```jsx
import { createStyled } from '@stitches/react';

const { styled } = createStyled({});

const Box = styled('div', {});
```

The reasons behind this is to:
- Easier and quicker access to styles, when not needing to pass custom configuration
- Closer to how other CSS in JS Libraries work

Note: the `css` function, which is also returned from `createStyled`, should probably also be available directly

What do you think?

**Comment by StephenHaney** [comment]
Love it! I find myself wanting this when I'm prototyping or doing a codesandbox – I think it's great for new users who are familiar with SC. This removes a speed bump to trying stitches out casually.

**Comment by peduarte** [comment]
This is in since `0.1.0`
